ios/Classes/PluginExtensions/SwiftOneginiPlugin+Auth.swift

```diff
@@ -33,28 +33,33 @@ extension SwiftOneginiPlugin: OneginiPluginAuthProtocol {
     }
 
     func authenticateUser(_ call: FlutterMethodCall, _ result: @escaping FlutterResult) {
-        guard let _arg = call.arguments as! [String: Any]? else { return; }
-        let _id = _arg["registeredAuthenticatorId"] as? String
-        if (_id != nil) {
-            // auth with provider
-            Logger.log("use provider for auth")
-            OneginiModuleSwift.sharedInstance.authenticateWithRegisteredAuthentication(_id, callback: result)
-        } else {
+        guard let arg = call.arguments as? [String: Any] else {
+            return result(SdkError(.methodArgumentNotFound).flutterError());
+        }
+
+        guard let profileId = arg["profileId"] as? String else {
+            return result(SdkError(.methodArgumentNotFound).flutterError());
+        }
+
+        guard let registeredAuthenticatorId = arg["registeredAuthenticatorId"] as? String else {
             // auth with pin
             Logger.log("use pin for auth")
-            OneginiModuleSwift.sharedInstance.authenticateUser(nil, callback: result)
+            OneginiModuleSwift.sharedInstance.authenticateUser(profileId, callback: result)
+            return
         }
+
+        // auth with provider
+        Logger.log("use provider for auth")
+        OneginiModuleSwift.sharedInstance.authenticateWithRegisteredAuthentication(profileId, registeredAuthenticatorId, callback: result)
     }
 
     func authenticateUserImplicitly(_ call: FlutterMethodCall, _ result: @escaping FlutterResult) {
         guard let arg = call.arguments as? [String: Any] else {
-            result(SdkError(.mappedMethodArgumentIsMissing).flutterError());
-            return;
+            return result(SdkError(.methodArgumentNotFound).flutterError());
```

Comment:
remove `;` at the end of lines everywhere in this PR.

Reply:
done